operate/services/protocol.py

```diff
@@ -841,15 +855,31 @@ def get_staking_params(self, staking_contract: str) -> t.Dict:
             staking_contract=staking_contract,
         )
 
-        return dict(
-            staking_contract=staking_contract,
-            agent_ids=agent_ids,
-            service_registry=service_registry,
-            staking_token=staking_token,
-            service_registry_token_utility=service_registry_token_utility,
-            min_staking_deposit=min_staking_deposit,
-            activity_checker=activity_checker,
-        )
+        output = {
+            "staking_contract": staking_contract,
+            "agent_ids": agent_ids,
+            "service_registry": service_registry,
+            "staking_token": staking_token,
+            "service_registry_token_utility": service_registry_token_utility,
+            "min_staking_deposit": min_staking_deposit,
+            "activity_checker": activity_checker,
+            "additional_staking_tokens": {},
+        }
+        try:
+            instance = staking_manager.dual_staking_ctr.get_instance(
+                ledger_api=self.ledger_api,
+                contract_address=staking_contract,
+            )
+            output["additional_staking_tokens"][
+                instance.functions.secondToken().call()
+            ] = instance.functions.secondTokenAmount().call()
+        except Exception:  # pylint: disable=broad-except # nosec
```

Comment:
@OjusWiZard I had to replace a `ContractLogicError` by an `Exception`. The problem is that on mainnet this was raised a `ValueError` as opposed to testnet, where it raised a `ContractLogicError`. The correct case should be the `ContractLogicError`, but the web3 library parses incomplete outputs from the RPC as `ValueError`.

Reply:
See also comment https://github.com/valory-xyz/olas-operate-middleware/pull/44/files#r1995347671

Reply:
Okay, I will update a comment on the source code.